### DISCUSSION TITLE
Store sessions in `state` directory by default?

### DISCUSSION THREAD

**Discussion by adrian5** [olimorris/persisted.nvim#124] [Ideas]
I'm wondering if sessions should be stored under `stdpath("state")` by default, rather than `stdpath("data")`.

* [stdpath](https://neovim.io/doc/user/builtin.html#stdpath())
* [base-directories](https://neovim.io/doc/user/starting.html#standard-path)

**Comment by olimorris** [comment]
I've seen justifications for both at various times which I've agreed with...but if I was starting this plugin again I would be choosing `stdpath("state")`.

Given there's a decent user base to this plugin and the option exists to overwrite the `save_dir`, I'll leave it as it is.
